src/components/operations/operation-details/ko/runtime/operation-console.html

```diff
@@ -247,9 +247,30 @@ <h3>Headers</h3>
                     <!-- /ko -->
                     <!-- ko if: !header.options || header.options.length === 0 -->
                     <div class="input-group">
+                        <!-- ko if: !header.secret -->
                         <input type="text" autocomplete="off" class="form-control form-control-sm" placeholder="value"
                             spellcheck="false" aria-label="Header value"
                             data-bind="event: { keyup: $component.updateRequestSummary }, textInput: header.value">
+                        <!-- /ko -->
+                        <!-- ko if: header.secret -->
+                        <div style="width: 100%;">
+                            <!-- ko if: !header.revealed() -->
+                            <input type="password" autocomplete="off" class="form-control form-control-sm password-input"
```

Comment:
`<input type="password" autocomplete="off"` - technically, you can re-use exiting markup and switch input type between "text" and "password" when toggling show/reveal. For that you can use `attr` binding. Example: `data-bind="attr: { type: myInputType }`.